site/src/components/Footer/Footer.tsx

```diff
@@ -24,16 +26,22 @@ export const Footer: React.FC<React.PropsWithChildren<FooterProps>> = ({
 }) => {
   const styles = useFooterStyles()
 
-  const githubUrl = `https://github.com/coder/coder/issues/new?labels=needs+grooming&body=${encodeURIComponent(`Version: [\`${buildInfo?.version}\`](${buildInfo?.external_url})
 
-<!--- Ask a question or leave feedback! -->`)}`
-  const discordUrl = `https://coder.com/chat?utm_source=coder&utm_medium=coder&utm_campaign=server-footer`
+  const githubUrl = 'https://github.com/coder/coder/issues/new?'
+  const body = encodeURIComponent(`Version: [\`${buildInfo?.version}\`](${buildInfo?.external_url})`)
+
+  const reportBugUrl = githubUrl + `labels=bug&body=${body}`
+  const enhancementUrl = githubUrl + `labels=enhancement&body=${body}`
+  const shareFeedbackUrl = githubUrl + `labels=feedback&body=${body}`
```

Comment:
This should work fine, but I think it's nice to build the URL objects. Here's one way you could do it. 
```
const buildURL = (label: string): URL => {
  const githubURL = new URL('https://github.com/coder/coder/issues/new')
  githubURL.searchParams.append("body", `Version: [\`${buildInfo?.version}\`](${buildInfo?.external_url})`)
  githubURL.searchParams.append("label", label)
  return githubURL
}
```

Reply:
Much better approach thank you.